atmat/atphysics/Radiation/ohmienvelope.m

```diff
@@ -24,6 +24,12 @@
 NumElements = length(ring);
 if nargin<3, refpts=1; end
 
+Wig=atgetcells(ring,'Bmax');
+Wigidx=find(Wig(:)==1);
+
+radindex(Wigidx) = false; % Erase wigglers from the radiative element list.

```

Comment:
```Matlab
radindex = radindex & ~Wig
```
is more efficient and easier to understand

Reply:
Thanks for the comment, I'll change this.